pycose/py.typed

```diff
@@ -0,0 +1 @@
+# Present to statisfy https://peps.python.org/pep-0561/#packaging-type-information
```

Comment:
```suggestion
# Present to satisfy https://peps.python.org/pep-0561/#packaging-type-information
```

Reply:
Instead of that I'd actually prefer something like https://github.com/python/mypy/blob/master/mypy/py.typed

Reply:
@letmaik ok, done.